esp-hal/src/usb_serial_jtag.rs

```diff
@@ -1,54 +1,103 @@
-//! # USB Serial JTAG peripheral driver
+//! USB Serial/JTAG Controller
 //!
-//! ## Overview
-//! The USB Serial JTAG peripheral driver provides an interface to communicate
-//! with the USB Serial/JTAG peripheral on ESP chips. It enables serial
-//! communication and JTAG debugging capabilities, allowing developers to
-//! interact with the ESP chip for programming, debugging, and data transfer
-//! purposes, can be also used to program the SoC's flash, read program output,
-//! as well as attach a debugger to a running program.
+//! The USB Serial/JTAG controller can be used to program the SoC's flash, read
```

Comment:
Really nice job! I love the style and flow of these docs - this is exemplary :star_struck: